pkg/api/v1/types.go

```diff
@@ -424,6 +428,21 @@ type RBDVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"  description:"rbd volume to be mounted with read-only permissions; see http://releases.k8s.io/HEAD/examples/rbd/README.md#how-to-use-it"`
 }
 
+// CinderVolumeSource represents a cinder volume resource in Openstack.
+// A Cinder volume must exist before mounting to a container.
+// The volume must also be in the same region as the kubelet.
+type CinderVolumeSource struct {
+	// volume id used to identify the volume in cinder
+	VolID string `json:"id" description:"Cinder volume id"`
+	// Required: Filesystem type to mount.
+	// Must be a filesystem type supported by the host operating system.
+	// Only ext3 and ext4 are allowed
+	FSType string `json:"fsType,omitempty" description:"FStype to which the volume will be formatted if not already of that type"`
```

Comment:
The description for this field should state that only ext3 and ext4 are supported.